README.md

```diff
@@ -195,3 +195,29 @@ As `Serilog.Sinks.Datadog.Logs` implements [Serilog.Sinks.PeriodicBatching](http
 
 From [Serilog.Sinks.Async documentation](https://github.com/serilog/serilog-sinks-async#serilogsinksasync---):	
 > Note: many of the network-based sinks (CouchDB, Elasticsearch, MongoDB, Seq, Splunk...) already perform asynchronous batching natively and do not benefit from this wrapper.
+
+
+## Development Environment 
+This repository includes a devcontainer configuration for VS Code. To use it:
+
+1. Install VS Code and Docker
+2. Open in VS Code with Dev Containers extension
```

Comment:
```suggestion
2. Open the configuration in VS Code with the Dev Containers extension.
```

Reply:
Ok if I change `the configuration` to `the project`? I think referring to the project (or repo) is more accurate.